tests/unit/test_fracture_network.py

```diff
@@ -314,6 +313,79 @@ def test_add_networks_preserve_tags(self):
         known_tags = np.array([tag1, tag1, tag2, tag2])
         self.assertTrue(np.all(together._edges[2] == known_tags))
 
+    def test_triangle_domain_no_fracs(self):
```

Comment:
Please provide a few lines at the top of the test describing what is being tested and what a failure of the test will tell us.

Reply:
I added few lines for each test to briefly describe it